product_docs/docs/biganimal/release/pricing_and_billing/index.mdx

```diff
@@ -24,7 +24,7 @@ This table shows the cost breakdown.
   
 ### Distributed high-availability pricing
 
-When distributed high-availability configurations are enabled, to calculate the full price for all resources used, multiply the number of vCPUs per instance by the number of data nodes configured. You aren't charged for the database price for witness nodes or groups in distributed high-availability configurations, just the infrastructure resources, such as compute.
+When distributed high-availability configurations are enabled, to calculate the full price for all resources used, multiply the number of vCPUs per instance by the number of data nodes configured. You aren't charged for the database price for witness nodes or groups in distributed high-availability configurations, just the infrastructure resources, such as compute. Distributed high-availability clusters are powered by EDB Postgres Distributed.
```

Comment:
Would it be worth making the EDB Postgres Distributed a like to say https://enterprisedb.com/docs/pgd ?